core/app/pdf/mno_enterprise/invoice_pdf.rb

```diff
@@ -34,6 +34,7 @@ def initialize(invoice)
         @data["customer_#{detail}".to_sym] = invoicable.respond_to?(detail) ? invoicable.send(detail) : nil
       end
 
+      @data[:deletion_notice] = invoicable.account_frozen && !invoicable.frozen_at.nil? && invoicable.frozen_at < invoice.ended_at
```

Comment:
Could extract this to a separate helper method?

Reply:
Added a method for `invoice`: `will_org_be_deleted?`... Couldn't come up with a better name (maybe just `deletion_notice?` - the PDF context is unclear)